tests/unit/models/test_redditors.py

```diff
@@ -18,18 +18,16 @@ def test_partial_redditors(self):
             list(self.reddit.redditors.partial_redditors(in_ids_list))
 
             assert mock_method.call_count == 1
-            assert mock_method.call_args[1]["params"]["ids"] == ",".join(
-                in_ids_list
-            )
+            assert mock_method.call_args[0][2]["ids"] == ",".join(in_ids_list)
```

Comment:
i just want to figure out why this test started failing as soon as I changed the signatures for get() and the other methods. I wonder why the old code even worked, because `mock_method.call_args[1]` did not exist, and it was a tuple, rather than a dictionary.

Reply:
Follow up: The code changed because the call for get() changed from keyword args to positionals.

Reply:
Please use keyword arguments pretty much everywhere unless a function takes only a single argument.